mopidy_tidal/ext.conf

```diff
@@ -3,3 +3,4 @@ enabled = true
 quality=LOSSLESS
 client_id=
 client_secret=
+playlist_cache_refresh_secs = 0
```

Comment:
Previously it looks like we used 300 as the default value (hardcoded). Should we keep it here or do we want to use 0 as the default value from now on?

Reply:
Looks like you have already elaborated on this in #91. Marking this as resolved.